### PR TITLE
chore: add my notes on chapter 1 of working the command line 

### LINKED ISSUE #22: chore: add my notes on chapter 1 of working the command line
I have some notes gathered throughout the reading that I'd like to add.

### PR DESCRIPTION
**Pull request by bpires**
I have some notes gathered throughout the reading that I'd like to add:
- permission details
- correct some typos

Fix #22

### CODE REVIEW

File: working-the-command-line/chapter1/index.md [hunk 1]
```diff
@@ -135,12 +143,13 @@ One way of searching your history is by pressing `Ctrl+R`
...
-bck-i-search: _ # Start typing your search. Once you find the command, press enter to execute it
+bck-i-search: _ # Start typing your search. 
+Once you find the command, press enter to execute it
 ```
 
-> NOTE: You can press `Ctrl+C` to exit our of search mode
+> NOTE: You can press `Ctrl+C` to exit out of search mode.
 
-Let’s say we run `ls -l` and press enter but then realize we wanted to run `ls -lt`. You could simple type it our or, you can do something like this:
+Let’s say we run `ls -l` and press enter but then realize we wanted to run `ls -lt`. You could simple type it out or, you can do something like this:
```
Comment (KobayashiComputing) [inline working-the-command-line/chapter1/index.md:152]: I would either remove the word "simple" completely, or change it to "simply" if you want to leave it in. Either way works. Also, I don't think the comma after the "or" is needed and should be removed.

File: working-the-command-line/chapter1/index.md [hunk 2]
```diff
@@ -82,7 +82,15 @@ There is more to the definition of "The Long Format" that you can find in the ma
 > - minute file last modified,
 > - and the pathname.
 
-### `-F`
+#### Understanding the permission details
+
+The first column, on the left, will present a series of letters and dashes that correspond to the file permissions.
+
+The first character is the file type (`-` for a regular file, `d` for a directory, and `l`for a symlink). Then, there are three groups of special characters regarding the permissions of the `user`, the `group`, and the `other` categories. Each trio is composed of the letters `r`, `w`, and `x`, which stands for read, write and execute permissions, respectively. When that category doesn't have a certain type of permission, the letter is replaced by a `-`.
```
Comment (KobayashiComputing) [inline working-the-command-line/chapter1/index.md:89]: Need to add a space between "'l'for a symlink" so that is reads "'l' for a symlink"

**Review by KobayashiComputing — CHANGES_REQUESTED** [review]

File: working-the-command-line/chapter1/index.md (hunk 2 above)
Comment (bpires) [reply working-the-command-line/chapter1/index.md:89]: Such great eye for detail!

**Review by KobayashiComputing — APPROVED** [review]
Looks good!

**Pull request merged**